UnofficialSteamAuthenticator/Models/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Windows.UI.Xaml.Media.Imaging;
+using SteamAuth;
+
+namespace UnofficialSteamAuthenticator.Models
+{
+    /// <summary>
+    /// Type for showing users in a list
+    /// </summary>
+    internal class User : ModelBase
+    {
+        public ulong SteamId { get; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+        public BitmapImage Avatar { get; set; }
+        private readonly SteamGuardAccount _steamGuardAccount;
+
+        public User(ulong steamId, string title, string content, BitmapImage avatar)
+        {
+            SteamId = steamId;
+            _steamGuardAccount = Storage.GetSteamGuardAccount(steamId);
+
+            Title = title;
```

Comment:
yes VS, "simplified". I just don't like things like "Title = title", it could look like it does nothing, the capitalisation I suppose does help.

I'd probably say everything and I'm sure I'm going to regret it when visual studio has 100s of suggestions for me.